### ISSUE TITLE
CREATE TABLE with UNIQUE columns

### ISSUE THREAD

**Issue by agershun** [agershun/alasql#169]
From issue https://github.com/meteor-stream/meteor-postgres/issues/239

![error](https://cloud.githubusercontent.com/assets/3205289/7336192/5f1c1048-ec22-11e4-9824-3aaa2841dc15.png)
In Todo List example, when i am creating table using the data types and options below. The $unique give me the above error. Any ideas why $unique is not working?
```js
var usersTable = {
    id: ['$number'],
    name: ['$string', '$notnull','$unique']
  };
  username.createTable(usersTable);
```
The problem with UNIQUE keyword in the column. TO be checked.

**Comment by agershun** [comment]
I added UNIQUE keyword in the list of column constraints, but the feature is not supported yet. I estimate that it takes about a month to add this functionality.

The minor syntax fix is available in develop branch or will be included in version 0.1.4 on May 11, 2015.

**Comment by agershun** [comment]
Solved. WIll be available in version 0.1.4.
